target/linux/ar71xx/base-files/etc/board.d/02_network

```diff
@@ -16,6 +16,9 @@ ar71xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "4:lan" "5:wan"
 		;;
+	ap531b0)
```

Comment:
This is redundant here, please drop it.
`ucidef_set_interfaces_lan_wan "eth0" "eth1"` is the default setting.